fix: update "General" docs (#893)
iresty/iresty.github.io: website/docs/general/committer-guide.md

- Answer questions in [issues](https://github.com/apache/apisix/issues) and [discussions](https://github.com/apache/apisix/discussions).

- Review open [pull requests](https://github.com/apache/apisix/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc).

- Improve the website.

- Improve documentation or write a blog post.

- Any other form of code or non-code contribution.

If you would like to contribute, let us know by sending an email to dev@apisix.apache.org!

## How to become an APISIX committer?

Anyone can be a contributor to an Apache project. Being a contributor means that you take an interest in the project and contribute to it in some way, ranging from asking sensible questions (which documents the project and provides feedback to developers) to working on new features and patches.

If you become a valuable contributor to the project you may be invited to become a Committer. A Committer is a term used at the ASF to signify someone who is committed to a particular project. It brings with it the privilege of write access to the project repository and resources.

More details could be found [here](https://community.apache.org/contributors/).

## Promotion

> The Apache APISIX community follows the Apache Community’s process on accepting a new committer. After a contributor participates in APISIX's community actively, (P)PMC and Committers will decide whether to invite the contributor to join Committers and (P)PMC.

Reviewer: ```suggestion
The Apache APISIX community follows the Apache Community’s process of accepting a new committer. After a contributor participates in APISIX's community actively, (P)PMC and Committers will decide whether to invite the contributor to join Committers and (P)PMC.
```

Reviewer 2: Actually the PMCs will go through a voting process after the nomination process, in which the PMCs will decide whether to invite this particular contributor to join as a Committer.

This has rarely anything to do with other Committers ( i.e. other Committers are not in any of the nominating, voting, or inviting process).

> 1. This process starts with a discussion and vote in @private. Only the current PMC members can nominate a new committer or a PMC member.

Reviewer 2: ```suggestion
1. This process starts with a discussion and vote in @private channel, where the current PMC members can nominate a new committer or a PMC member. Each new email in the discussion will extend the discussion for 72 hours. Please be patient while waiting for PMC members making their decision as a whole.
```

2. If the vote passes, an offer letter is sent to the contributor to become a committer with @private CC’ed.

> 3. The new committer can then sign the ICLA and apply for an Apache ID and email address.

Reviewer 2: Might as well add a link to the actual ICLA form.
```suggestion
3. The new committer can then sign the [ICLA](https://www.apache.org/licenses/contributor-agreements.html#clas) and apply for an Apache ID and email address.
```

> 4. The [Team](/team) page is updated with the new committer.

Reviewer 2: ```suggestion
4. Link your Apache account with your GitHub account via [GitBox](https://gitbox.apache.org/setup/). 

5. The [Team](/team) page is updated with the new committer. Examples can be found in [PR#605](https://github.com/apache/apisix-website/pull/605) and [PR#881](https://github.com/apache/apisix-website/pull/881). The new committer would need to fork and clone the `apisix-website` repository locally, and create a PR to add him/herself to the Committer section on the Team page manually.
```
